migrations/2023-11-07-135409_single-inbox/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table person drop column shared_inbox_url;
+alter table community drop column shared_inbox_url;
```

Comment:
i think your migration is missing something that moves what was previousy the shared inbox url into the inbox url field ? e.g. `update community set inbox_url = coalesce(shared_inbox_url, inbox_url)`

Reply:
I think he's handling that in the code migration, but if its doable in SQL, that would be preferred.

Reply:
I moved the ones for person and community to db migration, but anyway we still need a code migration for the site inbox (because it doesnt have any shared inbox field to copy from, and we dont know the domain in db).

Reply:
The site domain you can get from joining to the instance table I think.